### ISSUE TITLE
Blank code editor on user navigate back

### ISSUE THREAD

**Issue by ndresx** [codesandbox/codesandbox-client#2685]
# 🐛 bug report

## Description of the problem

1. Open sandbox
2. Click user avatar/name
3. Navigate back (from the user's profile page)

Observe the blank code editor. Clicking into it shows a new Untitled file, not the initial code. Saving it causes another error to appear (see attached screenshot below).

![image](https://user-images.githubusercontent.com/27507295/66340669-f8575900-e945-11e9-9a6b-d54f5e8eba60.png)

```
Uncaught Error: Cannot add node 1 because a node with that id is already in the Store.
```

Error message on save of the Untitled file:
![image](https://user-images.githubusercontent.com/27507295/66341227-1e312d80-e947-11e9-8319-f6cde4eef8cb.png)

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | ?
| Browser          | Google Chrome 
| Operating System | Windows 10

**Comment by dagolinuxoid** [comment]
@ndresx I was able to reproduce your issue - after navigating back I saw the same blank page in the source code area (all my tabs disappeared). I came up with the next solution: click on *explorer* menu and open needed files again - they should reappear in form of tabs as expected. I agree that it's a little bit tedious to do so and it would be nice if they preserve state but it's not the end of the world. (Linux, Google Chrome 77)
